src/components/standalone/monitoring/TrafficMonitor.vue

```diff
@@ -137,16 +137,13 @@ const hoursDatasets = computed(() => {
     <template v-else>
       <div class="flex flex-col gap-6 xl:flex-row">
         <div class="flex flex-col gap-6 sm:w-96">
-          <NeCard :title="t('standalone.real_time_monitor.daily_total_traffic')">
-            <SimpleStat>
-              <span v-if="data.total_traffic != 0">{{ byteFormat1024(data.total_traffic) }}</span>
-              <span v-else> N/A </span>
-            </SimpleStat>
-          </NeCard>
           <NeCard v-if="contains('client')" :title="t('standalone.dashboard.hostname')">
             <SimpleStat>
-              <p v-if="resolvedHostname != get('client')">{{ resolvedHostname }}</p>
-              <p class="[&:nth-child(2)]:text-lg">{{ get('client') }}</p>
+              <p v-if="resolvedHostname != get('client')">
+                {{ resolvedHostname }}
+              </p>
+              <p class="text-lg">probook445g10.nethesis.it</p>
```

Comment:
`probook445g10.nethesis.it`

:see_no_evil:

Reply:
Nothing here, keep scrolling :see_no_evil: